tests/internals.rs

```diff
@@ -364,3 +478,274 @@ fn test_declare_tx() {
     );
     assert_eq!(fee_transfer_info.l2_to_l1_messages, Vec::new());
 }
+
+#[test]
+fn test_deploy_account() {
+    let (general_config, mut state) = create_account_tx_test_state().unwrap();
+
+    let deploy_account_tx = InternalDeployAccount::new(
+        felt_to_hash(&TEST_ACCOUNT_CONTRACT_CLASS_HASH),
+        2,
+        TRANSACTION_VERSION,
+        Default::default(),
+        Default::default(),
+        Default::default(),
+        Default::default(),
+        StarknetChainId::TestNet,
+    )
+    .unwrap();
+
+    state.set_storage_at(
+        &(
+            general_config
+                .starknet_os_config()
+                .fee_token_address()
+                .clone(),
+            felt_to_hash(&TEST_ERC20_DEPLOYED_ACCOUNT_BALANCE_KEY),
+        ),
+        ACTUAL_FEE.clone(),
+    );
+
+    let (state_before, state_after) = expected_deploy_account_states();
+    if state != state_before {
+        println!(" left: {state:#?}");
+        println!("right: {state_before:#?}");
+        panic!();
```

Comment:
Wont be better to use an assert_eq!

Reply:
`assert_eq!()` prints the entire state in the same line, which has crashed my editor multiple times. This pretty-prints the states (using multiple lines). Since it already works I should probably leave an `assert_eq!()`.